ocs_ci/utility/utils.py

```diff
@@ -1072,8 +1072,11 @@ def get_ocp_version(seperator=None):
 
     """
     char = seperator if seperator else '.'
+    version = Version.coerce(
+        config.DEPLOYMENT['installer_version']
+    )
     return char.join(
-        config.DEPLOYMENT['installer_version'].split('.')[:2]
+        str(str(version.major) + str(version.minor))
```

Comment:
Shouldn't this be [str(version.major), str(version.minor)]?

Reply:
> Shouldn't this be [str(version.major), str(version.minor)]?

hmm, its working because for eg, "43" is also iterable. However I feel its better to stick to standard doc as the help for 'join' shows a list. I will change this.

Reply:
done